docs/how-to/craft/enable-classic-confinement.rst

```diff
@@ -3,18 +3,170 @@
+Store. If a user obtains your snap through other channels and they understand the risks,
+they can manually install it. Without approval, there's no way for them to obtain it
+from the Snap Store.
+
+`Submit your snap for review
+<https://snapcraft.io/docs/reviewing-classic-confinement-snaps>`_ to get approval for
+classic confinement.
+
+
+.. _how-to-enable-classic-confinement-identify-problems-linters:
 
 Identify problems with linters
 ------------------------------
 
-.. link to linting docs
+When you first enable classic confinement, you should run an appropriate :ref:`built-in
+linter <reference-linters>` to check for common errors and problems in the snap's files.
+
+
+.. _how-to-enable-classic-confinment-example-project-files:
+
+Example project files with classic confinement
+----------------------------------------------
+
+The following projects are sample solutions for combining classic confinement with
+specific plugins. These are only guidelines. Every case is different, so you may need to
+make other adjustments to your snap's build to completely confine it.
+
+
+Autotools plugin
+~~~~~~~~~~~~~~~~
+
+`autotools-classic-example
+<https://github.com/snapcraft-doc-samples-unofficial/autotools-classic-example>`_ is a
+project with a main part that uses the Autotools plugin.
+
+.. collapse:: autotools-classic-example project file
+
+    .. literalinclude:: code/example-classic-confinement-autotools.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Make plugin
+~~~~~~~~~~~
+
+`makefile-lib-example
+<https://github.com/snapcraft-doc-samples-unofficial/makefile-lib-example>`_ is a
+project with a main part that uses the Make plugin.
+
+.. collapse:: makefile-lib-example project file
+
+    .. literalinclude:: code/example-classic-confinement-make.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+CMake plugin
+~~~~~~~~~~~~
+
+`cmake-classic-example <https://github.com/snapcraft-docs/cmake-classic-example>`_ is a
+project with a main part that uses the CMake plugin.
+
+.. collapse:: cmake-classic-example project file
+
+    .. literalinclude:: code/example-classic-confinement-cmake.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Python plugin
+~~~~~~~~~~~~~
+
+`python-ctypes-example <https://github.com/snapcraft-docs/python-ctypes-example>`_ is a
+project with a main part that uses the Python plugin.
+
+.. collapse:: python-ctypes-example project file
+
+    .. literalinclude:: code/example-classic-confinement-python.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Patch Python ctypes to load system libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your app uses ctypes to access system libraries, it must be bundled with patched
+ctype files:
+
+1. Stage the system libraries in the main part. For the core22 base, the packages are:
```

Comment:
You could rephrase this to provide a some background on why Python 3.10 is used for core22:
```suggestion
1. Stage the system libraries in the main part. The core22 base uses python 3.10, so the packages are:
```

Reply:
[d49dfb3](https://github.com/canonical/snapcraft/pull/5345/commits/d49dfb3e5f8e44dca56906c51ed9a8f43e9f83ac)